antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -213,8 +197,11 @@ def _apply(self, study_data: FileStudy) -> CommandOutput:
         area_from = data["area_from"]
         area_to = data["area_to"]
 
-        self.parameters = self.parameters or {}
-        link_property = CreateLink.generate_link_properties(self.parameters)
+        properties = LinkProperties.model_validate(self.parameters or {})
```

Comment:
I think here we should parse according to the version, instead of exluding "by hand" the fields later.
If we parse the right version, it will be serialized as expected for this study version.